terraform/modules/hub/modules/ecs_asg/files/cloud-init.sh

```diff
@@ -11,20 +11,8 @@ function run-until-success() {
   done
 }
 
-CURL="curl"
```

Comment:
this is still in use down on line 91 in the new version

Reply:
nice spot :+1: ... fixed